CIP-0060/README.md

```diff
@@ -39,295 +43,221 @@ Music tokens on Cardano can be either NFTs or FTs and contain links to audio fil
+| ipi | String | "authors": [{"Mark Ronson":{"ipi:"157896357", "share":"25%"}}] |  Included in `song` within `authors` |
+| share | String | "authors": [{"Mark Ronson":{"ipi:"157896357", "share":"25%"}}] |  Included in `song` within `authors`.  Total percentage of all listed authors' shares MUST equal 100% |
+| metadata_language | String | "metadata_language": "en-US" | https://tools.ietf.org/search/bcp47 | Included in `song` |
+| country_of_origin | String | "country_of_origin": "United States" |  Included in `song` | 
+| language | String | "language": "en-US" | https://tools.ietf.org/search/bcp47 | Included in `song` |
+| derived_from | String | "derived_from" : "Some other work" |  Included in `song`|
 
 ### Examples ##
 
-#### Single ###
 ```
 {
-    "721":
-    {
-        "123da5e4ef337161779c6729d2acd765f7a33a833b2a21a063ef65a5":
-        {
-            "SickCity354":
-            {
-                "name": "SickCity354-Phil z'viel",
-                "image": "ipfs://QmQ13Cv9Wouf4rcwtNsuFhEeJVK9bEYjCYo6AN986takiu",
-                "music_metadata_version": 2,
+    "721": {
+        "<policyId>": {
+            "<assetName>": {
+                "name": "<releaseName>",
+                "image": "<mediaURL>",
+                "music_metadata_version": 3,
                 "release": {
-                    "release_type": "Single",
-                    "release_title": "C.H.I.L.L.",
-                    "distributor": "https://sickcity.xyz"
-                }
-                "files":
-                [
+                        "release_type": "<Single/Multiple>",
+                        "release_title": "<releaseTitle>",
+                        "distributor": "<distributor>"
+                          },                
+                "files": [
                     {
-                        "name": "C.H.I.L.L.",
-                        "mediaType": "audio/mp3",
-                        "src": "ipfs://QmfY4ugNdYYJxpbn5BnN8yt5EmiUCznexQoQi6RTEu9SuC"
+                        "name": "<fileName>",
+                        "mediaType": "<mimeType>",
+                        "src": "<mediaURL>",
                         "song": {
-                            "song_title": "C.H.I.L.L.",
-                            "song_duration": "PT3M6S",
-                            "track_number": 1,
-                            "mood": "chillout",
-                            "artists":
-                            [
-                                { "name": "Phil z'viel" }
+                            "song_title": "<songName>",
+                            "song_duration": "PT<minutes>M<seconds>S",
+                            "track_number": "<track#>",
+                            "mood": "<mood>",
+                            "artists": [
+                                {
+                                    "<artistName>": {
```

Comment:
Consider removing extra nesting, with artist name simple as a `name` key.  This will help reduce data-structure size overhead

Reply:
Considering readability on pool.pm, the flattened structure does not pass the name value.  While this is likely just a parsing issue, many users exclusively use this explorer to view their token data, and readability is the main consideration for this structure.  In any real use case, even if there are quite a few artists, any impacts on data use prove insignificant.  

I already have a music player that can pull from metadata, which can hopefully help remove the reliance on pool.pm for such tokens.  It is a non-issue to parse data using either structure. That being said, our discussions covered this and agreed to use the nesting for the reason above.  

Would you agree with our reasoning?

Reply:
cc @SmaugPool

Reply:
The value is passed, though it is quite cumbersome, as all keys and values are parsed in a single line.  The nested structure does seem to provide the best readability either way.

Reply:
I don't prefer the implicit semantics, but I understand the pool.pm consideration now that you've explicated it.  Maybe worth leaving a textual breadcrumb to indicate the rationale?

Reply:
In practical testing, it seems best practice to flatten the structure and have implementors adapt to the changes.  The nesting can create issues when entries lack optional fields while others will include them.  Flattening removes such conflicts.